docs/about/rationals.md

```diff
@@ -0,0 +1,75 @@
+# Rationals
+
+Before we recommend anything it goes through a process of evaluations.
+In short
+
+1. We only recommend usage of a features/function if it is release in a stable version of a least ONE browser
+2. We only recommend libraries if they offer an es module version
+
+::: warning
+We have ONE exception to this rule and that is 'bare modules'.
+This is such a powerful an prominent feature in the current js ecosystem so if you don't use it you basically need to implement everything yourself.
+We do not want you to do this so we need to embrace this as our only exeption for now.
+There is already a spec in the works which will bring bare modules via [import maps](https://github.com/WICG/import-maps) to the browser.
+So we hope this exeption will not stay forever.
+:::
+
+## Workflows
+
+Following there rules we strive to provide you with three development workflows.
+
+Each of the workflows tackles a specific task in your webcomponent/application.
+You are encouraged to freely switch between them depending on what you are working on.
+
+1. Developing workflow
+
+Use an up to date browser and develop with 0 tools => any webserver should do.
```

Comment:
we surely should have a section that explains that this may not always be archivable - e.g. depending on your situation you may want to "ditch" the native only and go webpack only... as open-wc we can however not recommend it but we understand if that's the reality you are in.